README.md

```diff
@@ -12,9 +12,11 @@ Builds of this image are available on (and should be pulled from) Docker Hub, wi
 |---|-----------|
 |marthoc/deconz:latest|Latest release of deCONZ, stable or beta|
 |marthoc/deconz:stable|Stable releases of deCONZ only|
-|marthoc/deconz:arch-version|Specific releases of deCONZ, use only if you wish to pin your version of deCONZ|
+|marthoc/deconz:version|Specific versions of deCONZ, use only if you wish to pin your version of deCONZ|
```

Comment:
Could you give an example for `version` - in my eyes it should be stated the `version` contrary to `latest` and `stable` is **not** ment literally.

Reply:
I'm not sure what you mean. In the previous iteration of this text "arch-version" was also not meant literally but I think that users understood what was being conveyed.

Reply:
That was what came to my mind first: _why should the be a tag with 'version' or 'arch-version' literally_

So I just wanted to place the opinion that examples could be helpful.